{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/settings/db.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+import dj_database_url
+
+DATABASES = {}
+DATABASES['default'] = dj_database_url.parse(os.environ.get('DATABASE_URL', 'postgres://'), conn_max_age=600)
+DATABASES['default']['ENGINE'] = 'django.db.backends.postgresql'
```

Comment:
Why is that? It overrides what was provided in `dj_database_url`, doesn't it?

Reply:
I does not. I just attach correct engine to the default setting without overriding others

Reply:
Is that so? What happens in that code, if you set `DATABASE_URL` to `mysql://`? 
The engine should be inferred from the connection string, or am I missing something?